src/data/logos.js

```diff
@@ -29,6 +29,11 @@ export const openSourceData = [
     logo: 'marianne',
     link: 'https://github.com/SocieteNumerique/pass-numerique',
   },
+  {
+    name: 'Bolt',
```

Comment:
IMHO it could even be fist

Reply:
Oh sure, I didn't know if I had to add it at the end or reorder all alphabetically like for companies.